utils/docker/vcluster/README.md

```diff
@@ -0,0 +1,199 @@
+    images and base image).
+
+## Running DAOS Nodes Container
+
+### Starting Containers with docker command
+
+Once the images are created, the containers could be directly started with docker with the following
+commands:
+
+```bash
+$ export DAOS_IFACE_IP=x.x.x.x
+$ docker run --detach --privileged --name=daos-server --hostname=daos-server \
+	--add-host "daos-server:$DAOS_IFACE_IP" --add-host "daos-admin:$DAOS_IFACE_IP" \
+	--add-host "daos-client:$DAOS_IFACE_IP" --volume=/sys/fs/cgroup:/sys/fs/cgroup:ro \
+	--volume=/dev/hugepages:/dev/hugepages  --tmpfs=/run --network=host \
+	daos-server:centos8
+$ docker run --detach --privileged --name=daos-agent --hostname=daos-agent \
+	--add-host "daos-server:$DAOS_IFACE_IP" --add-host "daos-admin:$DAOS_IFACE_IP" \
+	--add-host "daos-client:$DAOS_IFACE_IP" --volume=/sys/fs/cgroup:/sys/fs/cgroup:ro \
+	--tmpfs=/run --network=host daos-agent:centos8
+$ docker run --detach --privileged --name=daos-client --hostname=daos-client \
+	--add-host "daos-server:$DAOS_IFACE_IP" --add-host "daos-admin:$DAOS_IFACE_IP" \
+	--add-host "daos-client:$DAOS_IFACE_IP" --volume=/sys/fs/cgroup:/sys/fs/cgroup:ro \
+	--tmpfs=/run --network=host daos-client:centos8
+```
+
+The value of the `DAOS_IFACE_IP` shall be replaced with the one of the network interface which was
+provided when the images have been built.
+
+Once started, the DAOS server waits for the administrator to format the system.
+This can be done using the following command:
+
+```bash
+$ docker exec daos-admin dmg -i storage format
+```
+
+Upon successful completion of the format, the storage engine is started, and pools
+can be created using the daos admin tool.  For more advanced configurations and usage refer to the
+section [DAOS Tour](https://docs.daos.io/QSG/tour/).
+
+
+### Starting Containers with docker-compose command
+
+From a local tree, a more straightforward way to start the containers could be done with
+`docker-compose` and the following commands:
+
+```bash
+$ cd utils/docker/vcluster
+$ docker-compose up --detach
+```
+
+!!! note
+    Before starting the containers with `docker-compose`, the IP address of the network interface,
+    which was provided when the images have been built, shall be defined in the Docker
+    Compose environment file `utils/docker/vcluster/.env`.
+
+As with the docker command, the system shall be formatted, pools created, etc..
+
+
+### Managing Virtual Docker cluster with daos-cm.sh
+
+From a local tree, the bash script `utils/docker/daos-cm.sh` could be used to start the containers
+and setup a simple DAOS system composed of the following elements:
+
+- 1 DAOS pool of 10GB (i.e. size of the pool is configurable)
+- 1 DAOS POSIX container mounted on /mnt/daos-posix-fs
+
+This script could also be used to respectively stop and monitor the containers.
+
+More details on the usage of `daos-cm.sh` command could be found with running the following command:
+
+```bash
+$ bash utils/docker/daos-cm.sh --help
```

Comment:
This path is wrong, and can we just set execute permissions on the file?  Following the above examples...
```suggestion
$ cd utils/docker/vcluster
$ ./daos-cm.sh --help
```

Reply:
It is working from anywhere: there si a cd just before the switch-case.
https://github.com/knard-intel/daos/blob/8ccbf36ca07b92b342baba440f407062e54862fb/utils/docker/vcluster/daos-cm.sh#L224

Reply:
It does work from anywhere, but the script will not work if you call it as utils/docker/daos-cm.sh as it's not in the utils/docker directory.

Reply:
My bad, I did not understood your remark.
fixed with commit https://github.com/daos-stack/daos/pull/7556/commits/61862ea6fb4e6a26720ca90f8a71ee4ea8c1ad32